cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+  - Beatrice Anihiri
+  - Denis Kalinin
+  - Evgenii Lisitskii
+  - George APEX Pool
+  - George Flerovsky
+  - George Humphreys
+  - Hernán Rajchert
+  - Jack Rousian
+  - Joshua Marchand
+  - Ken Fritschy
+  - Ken-Erik Ølmheim
+  - Leo H
+  - Marcel Baumberg
+  - Martynas Kazlaukas
+  - Michael Chappell
+  - Mircea Hasegan
+  - Nicolas Ayotte
+  - Rhys Bartels-Waller
+  - Robert Phair
+  - Rodolpho Ribeiro
+  - Steven Johnson
+  - Teodora Sevastru Lunn
+  - Thomas Lindseth
+  - Thomas Upfield
+  - Vladimir Kalnitsky
+
+</details>
+
+
+## Problem
+<!-- A more elaborate description of the problem and its context. This section should explain what motivates the writing of the CPS document. -->
+
+The core motivation of this document is to catalog the current state of Cardano's wallet connectors, discuss their flaws and identify keys concerns for future connector authors to be aware of.
+We hope that by discussing the issues we will inspire the next generation of Cardano wallet connectors so that the ecosystem can grow beyond it's first iterations.
+
+Ineffective dApp-connectors can cause a range of issues for wallets, dApp developers and thus users.
+Due to the nature of these connections there can often be unforeseen impacts of small design decisions.
+
+For users, connectors should be widely supported across platforms, secure and decentralized.
+For dApps, connectors should be expressive and extendable.
+For wallets, connectors should be secure, extendable and should not expect wallet's to go beyond their standard activities.
+
+### Core Concerns
+
+#### 1. Secure connection
+Security should remain of paramount concern within web 3.
+Connection standards must strive to above all not compromise the security of wallets or dApps.
+Connection standards should be aware of potential impacts of standard security vulnerabilities, such as man-in-middle attacks.
+Furthermore, standards should seek to be censorship resistant, by an ecosystem offering multiple connection standards we move in this direction.
+
+#### 2. Secure API
+The security of the API itself should again remain paramount.
+This means that no secret information should ever be allowed to leave the wallet.
+Furthermore if an operation requires the wallet to use a secret, the user should always be made aware. 
+
+#### 3. Range of supported connection
+Connection standards should ideally support a wide range of wallets and dApp platforms.
+For example, connection which are accessible via desktop or mobile devices should be prioritized more so than those which only support one platform.
+
+#### 4. Expressive API
+APIs should allow for an expressive range of information to be exchanged.
+Generally communication should be allowed two ways.
+
+#### 5. Versioned
+Connection standards and APIs should be explicitly versioned to clearly allow upgrades.
```

Comment:
AFAIK CIPs are not versioned, right?

Reply:
Thanks @rphair 

@klntsky CIPs should include description of how they should be versioned, we added this a few months back.